robot/src/main/java/us/ilite/robot/Robot.java

```diff
@@ -121,6 +135,13 @@ public void autonomousInit() {
         mLoopManager.setRunningLoops(mDrive);
         mLoopManager.start();
 
+        // mCommandQueue.setCommands(new FollowTrajectory(trajectory, mDrive, true));
+//        mCommandQueue.setCommands(new CharacterizeDrive(mDrive, false, false));
+        mCommandQueue.setCommands(
```

Comment:
Please remove any temporary stuff that was inserted for testing (there will be a merge error here otherwise since we don't use `mCommandQueue` anymore)